lib/experimental/class-gutenberg-rest-comment-controller.php

```diff
@@ -303,6 +303,11 @@ public function create_item( $request ) {
 			$prepared_comment['comment_content'] = '';
 		}
 
+		// Include block comment metadata into check_is_comment_content_allowed [backport].
+		if ( ! empty( $request['meta'] ) && isset( $request['meta']['_wp_block_comment_status'] ) ) {
```

Comment:
```suggestion
		if ( isset( $request['meta']['_wp_block_comment_status'] ) ) {
```

This simplification should not cause any errors.